Xamarin.Forms.ControlGallery.iOS/AppDelegate.cs

```diff
@@ -169,12 +169,13 @@ public override bool FinishedLaunching(UIApplication uiApplication, NSDictionary
 					//	e.NativeView.AccessibilityIdentifier = e.View.StyleId;
 				}
 
-				if (e.NativeView != null)
-				{
-					var view = e.NativeView;
-					var tapGestureRecognizer = new UITapGestureRecognizer(() => Reset("")) { NumberOfTapsRequired = 5 };
-					view.AddGestureRecognizer(tapGestureRecognizer);
-				}
+				// Used shakegesture insted https://github.com/xamarin/Xamarin.Forms/issues/3724 
```

Comment:
just delete this code instead of commenting it out

Reply:
done